### PR TITLE
Blog Post: Will Clark LTS

### PR DESCRIPTION
**Pull request by dergigi**
Reviewed, edited, and approved.

Ready to be merged after sanity check.

---

Preview:

- Build preview: [/blog/will-clark-receives-lts-grant-for-bitcoin-core](https://os-website-git-will-clark-lts-open-sats.vercel.app/blog/will-clark-receives-lts-grant-for-bitcoin-core)
- Social preview: [day](https://github.com/OpenSats/website/assets/109058/75611d3f-fd46-4992-9e78-c204a2b332ca), [night](https://github.com/OpenSats/website/assets/109058/d88c55e2-2b90-46f6-a85c-7e8fd08c581d)

### CODE REVIEW

File: data/blog/will-clark-receives-lts-grant-for-bitcoin-core.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+draft: false
+authors: ['dergigi']
+images: ['/static/images/blog/16-will-clark.jpg']
+summary: "OpenSats Welcomes Will Clark as an LTS Grantee."
+---
+
+OpenSats is proud to provide long-term support for [Will
+Clark](https://github.com/willcl-ark). Will is a
+[Chaincode](https://chaincode.com/) alumnus and
+[Quala](https://builders.btrust.tech/) mentor who has been contributing to
```
Comment (abitcoinperson) [inline data/blog/will-clark-receives-lts-grant-for-bitcoin-core.mdx:14]:
```suggestion
[Qala](https://builders.btrust.tech/) mentor who has been contributing to
```

File: data/blog/will-clark-receives-lts-grant-for-bitcoin-core.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+draft: false
+authors: ['dergigi']
+images: ['/static/images/blog/16-will-clark.jpg']
+summary: "OpenSats Welcomes Will Clark as an LTS Grantee."
+---
+
+OpenSats is proud to provide long-term support for [Will
+Clark](https://github.com/willcl-ark). Will is a
+[Chaincode](https://chaincode.com/) alumnus and
+[Qala](https://builders.btrust.tech/) mentor who has been contributing to
```
Comment (willcl-ark) [inline data/blog/will-clark-receives-lts-grant-for-bitcoin-core.mdx:14]:
```suggestion
[BTrust (formerly Qala)](https://builders.btrust.tech/) mentor who has been contributing to
```

**Review by willcl-ark — APPROVED** [review]
LGTM, thanks! I left one final nit which you could choose to take, as I think the rebrand form Qala to BTrust is pretty complete.

**Review by Arvin21M — APPROVED** [review]
Content looks clean. All of the links work. There are two URLs for "contributing fixes" (https://bitcoinacks.com/?flt0_author_contains=willcl-ark) and "PR review" (https://bitcoinacks.com/?flt0_reviewer_contains=willcl-ark) that are coming back with a "NET::ERR_CERT_DATE_INVALID" error due to no SSL cert apparently. The warning can be bypassed to access these two pages, however.

**Comment by moteanup24** [comment]
Do we want to standardize capitalization of Bitcoin in tags? It is capitalized in the project files.

**Comment by dergigi** [comment]
> Do we want to standardize capitalization of Bitcoin in tags? It is capitalized in the project files.

@moteanup24 tags for projects and tags for blog posts are different; the blog engine transforms everything to lower-case anyway, so it doesn't matter in this case.

---

Thanks for the sanity check @Arvin21M, nothing we can do about the SSL issues of bitcoinacks.

**Pull request merged**